### PR TITLE
install pico extras

### PR DESCRIPTION
**Pull request by phoddie**
Pico I2S support is in extras. Needed for audio playback.

### CODE REVIEW

File: src/toolbox/setup/pico.ts [hunk 1]
```diff
@@ -11,10 +11,12 @@ export default async function(): Promise<void> {
   const OS = platformType().toLowerCase()
   const PICO_BRANCH = "1.5.0"
   const PICO_SDK_REPO = 'https://github.com/raspberrypi/pico-sdk'
+  const PICO_EXTRAS_REPO = 'https://github.com/raspberrypi/pico-extras'
   const PICO_EXAMPLES_REPO = 'https://github.com/raspberrypi/pico-examples'
   const PICOTOOL_REPO = 'https://github.com/raspberrypi/picotool'
   const PICO_ROOT = process.env.PICO_ROOT ?? filesystem.resolve(INSTALL_DIR, 'pico')
   const PICO_SDK_DIR = filesystem.resolve(PICO_ROOT, 'pico-sdk')
+  const PICO_EXTRAS_PATH = filesystem.resolve(PICO_ROOT, 'pico-extras')
```
Comment (HipsterBrown) [inline src/toolbox/setup/pico.ts:19]: Does the path need to be set as an environment variable to get the expected tooling?

**Review by HipsterBrown — COMMENTED** [review]
Thanks for adding this. Just a question about how the "extras" are shared with the rest of the system after cloning the repo.

File: src/toolbox/setup/pico.ts (hunk 1 above)
Comment (phoddie) [reply src/toolbox/setup/pico.ts:19]: No. CC-ing @mkellner just in case.

Comment (mkellner) [reply src/toolbox/setup/pico.ts:19]: We default to `$(HOME)/pico/pico-extras`.
If `PICO_EXTRAS_DIR` environment variable is set, it will override that.

Comment (phoddie) [reply src/toolbox/setup/pico.ts:19]: Glad @mkellner knows how it works. ... thanks for catching that @HipsterBrown.

**Comment by phoddie** [comment]
Renamed PICO_EXTRAS_PATH to PICO_EXTRAS_DIR for consistency and set PICO_EXTRAS_DIR in the environment.

**Comment by phoddie** [comment]
Thanks for getting this merged and published. It puts xs-dev slightly ahead of the next Moddable SDK. 🎉

**Pull request merged**